src/api-reference/expense/travelallowance/travelallowance-days.md

```diff
@@ -0,0 +1,192 @@
+---
+title: TravelAllowance Days
+layout: reference
+---
+
+This Travel Allowance API fetches all the fixed allowance days in an expense report. Clients using this API can determine whether an allowance is a full day or partial day allowance.
+
+> **Limitations**: This API is available only for Finland customers.
```

Comment:
Is this also only available in the EMEA data center?

Reply:
No. It's in EMEA and US